app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -200,6 +200,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 * Whether items shown in the screensaver are required to have an age rating set.
 		 */
 		var screensaverAgeRatingRequired = booleanPreference("screensaver_agerating_required", true)
+
+		/**
+		 * Delay when starting video playback after loading the video player.
+		 */
+		var videoStartDelay = longPreference("video_start_delay", 750)
```

Comment:
Could default to just `0`. There really is no reason to delay playback by default.

Reply:
This has been updated to 0 now.